packages/MarkdownEditor-Tests.package/MarkdownCommentBlockTests.class/instance/testCommentStart0To3Indentation.st

```diff
@@ -0,0 +1,6 @@
+tests - comment
+testCommentStart0To3Indentation
+
+	(Dictionary newFrom: {'<!--'->1.' <!--'->2. '  <!--'->3. '   <!--'->4}) keysAndValuesDo: [:commentLine :start |
+		self assert: start equals: (MarkdownCommentBlock newFrom: commentLine) commentStart]
```

Comment:
I'd prefer to put the comment block into an own variable to avoid overlooking `commentStart` and making the line shorter.

Also, please put `keysAndValuesDo:` on a new line